docker/images/n8n/n8n-task-runners.json

```diff
@@ -11,9 +11,9 @@
 				"N8N_RUNNERS_N8N_URI",
 				"N8N_RUNNERS_MAX_PAYLOAD",
 				"N8N_RUNNERS_MAX_CONCURRENCY",
-				"N8N_RUNNERS_SERVER_ENABLED",
-				"N8N_RUNNERS_SERVER_HOST",
-				"N8N_RUNNERS_SERVER_PORT",
+				"N8N_RUNNERS_RUNNER_HEALTH_CHECK_SERVER_ENABLED",
+				"N8N_RUNNERS_RUNNER_HEALTH_CHECK_SERVER_HOST",
+				"N8N_RUNNERS_RUNNER_HEALTH_CHECK_SERVER_PORT",
```

Comment:
As mentioned in the sister PR, these options take a repetitive infix `_RUNNER_` but the ones above don't? I'd be for dropping this infix.